src/components/BlogContextMenu.js

```diff
@@ -20,7 +20,12 @@ class BlogContextMenu extends React.Component {
   }
   
   clickEdit(post) {    
-    csrfFetch("/cube/blogsrc/" + post._id)
+    csrfFetch("/cube/blogsrc/" + post._id, {
+        method: "GET",
+        headers: {
+          'Content-Type': 'application/json'
```

Comment:
I don't think this is necessary. `Content-Type` specifies the content _you_ are sending, not the content you expect to receive. That's the `Accept` header.